level-ws.js

```diff
@@ -49,17 +53,7 @@ WriteStream.prototype._flush = function () {
   if (self.destroyed || !buffer) return
 
   self._buffer = []
-
-  self._db.batch(buffer.map(function (d) {
-    return {
-      type: d.type || self._options.type,
-      key: d.key,
-      value: d.value,
-      keyEncoding: d.keyEncoding || self._options.keyEncoding,
-      valueEncoding: (d.valueEncoding || d.encoding ||
```

Comment:
Not sure we should fully remove these options. Not having defaults for them is OK, but maybe we should pass-through any additional options?

Reply:
Yah, that's fine I think. Then we support `levelup@1` as well. Need to update `UPGRADING.md` as well with this information.

Reply:
Agreed. I wasn't thinking. I think we should restore encodings the way they were.